client/SFTPClientGUI/SFTPClientGUI.py

```diff
@@ -0,0 +1,351 @@
+import os
+import sys
+import time
+import stat
+import shutil
+
+import king_phisher.client.plugins as plugins
+
+from gi.repository import Gtk
+from gi.repository.GdkPixbuf import Pixbuf
+
+import boltons.strutils
+import boltons.timeutils
+
+class Plugin(plugins.ClientPlugin):
+	authors = ['Josh Jacob']
+	title = 'SFTP Client GUI'
+	description = """
+	How many foos could fu foo is fu fooed fu foos on foo.
+	"""
+	homepage = 'https://github.com/securestate/king-phisher'
+
+	def initialize(self):
+		module = sys.modules['king_phisher.client.application']
+		self.temp = module.KingPhisherClientApplication.do_sftp_client_start
```

Comment:
No, this is **not** how you connect signals, use the [signal_connect](http://king-phisher.readthedocs.io/en/latest/king_phisher/client/plugins.html#king_phisher.client.plugins.ClientPlugin.signal_connect) method.